perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -1275,16 +1275,42 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 	 * @param attribute AttributeDefinition object
 	 * @return AttributeDefinition Created AttributeDefinition
 	 */
+	/*#
+	 * Creates AttributeDefinition
+	 *
+	 * @param friendlyName String friendlyName
+	 * @param namespace String namespace
+	 * @param description String description
+	 * @param type String type
+	 * @param displayName String displayName
+	 * @param writable Boolean writable
```

Comment:
Writable flag is actually set on attribute retrieval based on users rights for specified attribute. It might be confusing having this option in API - user could think its some generic option. We should add it between ignored properties on read by deserializer instead (project ``perun-base``, package ``...rpc.deserializer``).